docs/breez-sdk/src/guide/getting_started.md

```diff
@@ -10,6 +10,7 @@ Before you begin, make sure to read the [repository README](https://github.com/b
 - **[Listening to events](/guide/events.md)**
 - **[Adding logging](/guide/logging.md)**
 - **[Custom configuration](/guide/config.md)**
+- **[Testing and development](/guide/testing.md)**
```

Comment:
Is it worth putting this after "Installing the SDK"? Or additionally writing a short sentence above to guide them to this page? (so they read is first)

Reply:
Moved to after "Installing the SDK"